nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -131,6 +131,15 @@
       and <option>services.limesurvey.virtualHost</option> options.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The setopt declarations will be evaluated at the end of zshrc, so any code in interactiveShellInit,
```

Comment:
I guess it's better to say `<literal>/etc/zshrc</literal>` here. Should've mentioned that previously, sorry.